docs/installation/rhel.md

```diff
@@ -12,150 +12,178 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of RHEL:
 
-- [*Red Hat Enterprise Linux 7 (64-bit)*](#red-hat-enterprise-linux-7-installation)
-- [*Red Hat Enterprise Linux 6.6 (64-bit)*](#red-hat-enterprise-linux-66-installation) or later
+- Red Hat Enterprise Linux 7 
+- Red Hat Enterprise Linux 6.6 or later
```

Comment:
Is the minimum on RHEL 6.6, or 6.5 (just as CentOS)? I know I mentioned we *recommend* 6.6, but not sure what the minimum is on RHEL